ui.tests/test-module/specs/fileinput/fileinput.runtime.spec.js

```diff
@@ -187,15 +201,22 @@ describe("Form with File Input - Prefill & Submit tests", () => {
                     cy.stub(win, 'open'); // creating a stub to check file preview
                 }
             });
-            const fileName = "empty.pdf";
+
             // attach the file
-            cy.get(fileInput.selector).attachFile(fileName);
+            cy.get(fileInput.selector).attachFile(fileInput.fileNames);
+            if(fileInput.multiple)
```

Comment:
This `if` would never execute, since `multiple` key does not exist for fileInput. Earlier, I thought this to be a HTML element, hence I suggested multiple to you. Can you fix this ?

Reply:
it will execute, I already added a boolean, see this: https://github.com/adobe/aem-core-forms-components/pull/811/files#diff-f650cd9d018f72906effe59ad366addeca22f4cc5637ebf84c1db50cd7cdedd7R194